lib/Magento2Connector/ApiHandler.php

```diff
@@ -37,25 +38,28 @@ public static function registerApiHandler(GenericEvent $event)
             $target = $event->getTarget();
 
             /** @var CrudListenerInterface $listener */
-            $listener = ListenerFactory::getFactory($target);
+            $listener = ListenerFactory::getListener($target);
 
-            if($listener === null) {
+            if ($listener === null) {
                 return;
             }
 
-            switch($event->getName()) {
+            switch ($event->getName()) {
                 case self::POST_UPDATE:
                     $listener->onPostUpdate($event);
                     break;
                 case self::POST_DELETE:
                     $listener->onPostDelete($event);
                     break;
             }
-        }
-        catch (ListenerNotFoundException $listenerNotFoundException) {
-            $apiLogger->info(sprintf("There is no handler for object type %s.   Register listener for this case.", get_class($target)));
-        }
-        catch (\RuntimeException $runtimeException) {
+        } catch (ListenerNotFoundException $listenerNotFoundException) {
+            $apiLogger->info(
+                sprintf(
+                    "There is no handler for object type %s.   Register listener for this case.",
```

Comment:
Pleas remove unnecessary spacing before phrase "Register listener for this case. "